.ci/env/tbb.sh

```diff
@@ -0,0 +1,26 @@
+sudo apt-get update
+sudo apt-get install build-essential gcc gfortran cmake -y
+git clone https://github.com/oneapi-src/oneTBB.git
+CoreCount=$(lscpu -p | grep -Ev '^#' | wc -l)
```

Comment:
since `CoreCount=$(lscpu -p | grep -Ev '^#' | wc -l)` code is taken from `.ci/env/openblas.sh` so taking suggestion from intel team would be better. @napetrov please suggest if changes are required?

Reply:
There is no strong preference. We can try nproc, but align across other scripts.